src/Model/Interactive.php

```diff
@@ -58,27 +59,55 @@ class Interactive extends DataObject {
         'TrackViews'        => 'Varchar(16)',
 
         'CompletionElement'   => 'Varchar(64)',         // what element needs clicking to be considered a 'complete' event
+    ];
 
-	);
-
-	private static $has_one = array(
+	private static $has_one = [
 		'InternalPage'		=> 'Page',
 		'Campaign'			=> InteractiveCampaign::class,
 		'Image'				=> Image::class,
-    );
+    ];
 
     private static $owns = [
         'Image',
     ];
 
-    private static $extensions = array(
+    private static $extensions = [
         InteractiveLocationExtension::class,
         Versioned::class
-    );
+    ];
 
     private static $tracker_type = 'Local';
 
-    private static $summary_fields = array('Title', 'Clicks', 'Impressions', 'Completes');
+    private static $summary_fields = ['Title', 'Clicks', 'Impressions', 'Completes'];
+
+    public function populateDefaults()
```

Comment:
Any reason to not use `private static $defaults` for these fields?

Reply:
Mostly for consistency with all the other configurable fields. Have changed the couple fields that can be.

Reply:
Looks like all of them can be - remember you can set the defaults static in yml

Reply:
It's a wonder I haven't encountered this before! Done.